src/scenes/gameplay-scene/gameplay-flow-manager.ts

```
@@ -0,0 +1,370 @@
+    private startGameTime(): void {
+        this.startTime = Date.now();
+    }
+
+    private startPuzzleTime(): void {
+        this.puzzleTime = Date.now();
+    }
+    // #endregion
+
+    public dispose(): void {
+        this.isDisposing = true;
+        this.eventListeners.forEach(unsubscribe => unsubscribe());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)